source/Octostache/VariableDictionary.cs

```diff
@@ -326,5 +327,16 @@ public void Add(string key, string? value)
         {
             Set(key, value);
         }
+
+        /// <summary>
+        /// Adds a custom extension function
+        /// </summary>
+        /// <param name="name"></param>
+        /// <param name="func"></param>
+        public void AddExtension(string name, Func<string?, string[], string?> func)
+        {
+            // Naming conflicts with BuiltInFunctions?
```

Comment:
This has been handled in `TemplateEvaluator`, so I don't think we need this comment any more